pkg/kind/kind.go

```diff
@@ -26,14 +26,15 @@ import (
 	"knative.dev/kn-plugin-quickstart/pkg/install"
 )
 
-var kubernetesVersion = "v1.23.3"
+var kindNodeImage string
 var clusterName string
 var kindVersion = 0.11
 
 // SetUp creates a local Kind cluster and installs all the relevant Knative components
-func SetUp(name string) error {
+func SetUp(name string, image string) error {
 	start := time.Now()
 	clusterName = name
+	kindNodeImage = image
```

Comment:
If we switch the default to `""` (per above comment), we probably only want to do if a non empty value is passed, so I think we'd want something like

```go
if image != "" {
    nodeImage = image
}
```